example/ck_tile/03_gemm/gemm_basic.hpp

```diff
@@ -10,11 +10,19 @@
 #include "ck_tile/ops/gemm.hpp"
 #include "ck_tile/ops/epilogue.hpp"
 
+struct GemmFp16
+{
+};
+
+struct GemmBf16
+{
+};
+
 template <typename DataType>
 struct GemmBasicTypeConfig;
```

Comment:
Please add documentation. Describe what this class defines ( supported data type configurations).